### ISSUE TITLE
IXP DB support to collect data from peers available at an IXP

### ISSUE THREAD

**Issue by tking** [peering-manager/peering-manager#481]
The IXP database https://ixpdb.euro-ix.net/en/ provides a lot of information about peers available from an IXP. The data is more complete compared to PeeringDB as the data is maintained by the IXP instead of individual peers.

IXP DB also provides an API.

**Comment by gmazoyer** [comment]
Do you know if IXP DB's API is freely available like PeeringDB's?

**Comment by tking** [comment]
Yes, the API is freely available: https://api.ixpdb.net/docs/On 28 Aug 2021 17:42, Guillaume Mazoyer ***@***.***> wrote:
Do you know if IXP DB's API is freely available like PeeringDB's?
